python/pyproject.toml

```diff
@@ -16,7 +16,7 @@ python-dotenv = "1.0.0"
 regex = "^2023.6.3"
 openapi_core = "^0.18.0"
 prance = "^23.6.21.0"
-pydantic = "<2"
+pydantic = ">2"
```

Comment:
I can't comment on it above, but can we please upgrade the `semantic-kernel` version to "0.4.1.dev"?

Reply:
We chatted offline. No need to update now.